src/bitcoin-cli.cpp

```diff
@@ -85,7 +85,7 @@ static bool AppInitRPC(int argc, char* argv[])
         }
 
         fprintf(stdout, "%s", strUsage.c_str());
-        return false;
+        if(argc < 2) return false; else exit(0);
```

Comment:
Also, don't combine 4 separate lines into one with no braces and nothing. Not only does it violate our coding guideline aesthetically; it also introduced really nasty bugs in the past.